lib/cocoapods/installer.rb

```diff
@@ -498,6 +498,7 @@ def install_pod_sources
             previous_version = sandbox.manifest.version(spec.name)
             has_changed_version = current_version != previous_version
             current_repo = analysis_result.specs_by_source.detect { |key, values| break key if values.map(&:name).include?(spec.name) }
+            current_repo &&= Pod::TrunkSource::TRUNK_REPO_NAME if current_repo.name == Pod::TrunkSource::TRUNK_REPO_NAME
             current_repo &&= current_repo.url || current_repo.name
```

Comment:
I need to do an if else here, but rubocop won't let me. How is this suppose to be done?

Reply:
I guess onelining was the solution